cloud/jenkins/pxc_operator_minikube.groovy

```diff
@@ -188,7 +188,7 @@ pipeline {
             description: 'PXC logcollector image: perconalab/percona-xtradb-cluster-operator:main-logcollector',
             name: 'IMAGE_LOGCOLLECTOR')
         string(
-            defaultValue: 'v1.14.8',
+            defaultValue: 'latest',
```

Comment:
I doubt that it's a good call
We need to keep version stated explicitly.

Reply:
Why? Anyway we check it during release?

Reply:
The explicit version will give us confidence about what and where was tested. 
Otherwise we may trap into unknown territory and spend a bit more time on getting the truth.